nomad/jitsi_packs/packs/jitsi_cloudprober/templates/_cloudprober_config.tpl

```diff
@@ -321,7 +321,7 @@ probe {
   validator {
       name: "status_code_2xx"
       http_validator {
-          success_status_codes: "200-299"
+          success_status_codes: "200-299,429"
```

Comment:
don't think this is valid?
```suggestion
          success_status_codes: "200-299"
```

Reply:
oops, this was intended to just be for vault